### PR TITLE
Adds version in binary

### PR DESCRIPTION
**Pull request by EmileVauge**
This PR adds a version number created during build time and injected into binary.
```sh
$ traefik --version
37438a6395324e9452f8f6f173f60ba33ce4bafe built on the 2015-10-14_08:02:45PM
```

### CODE REVIEW

File: traefik.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/mailgun/oxy/cbreaker"
 	"github.com/mailgun/oxy/forward"
 	"github.com/mailgun/oxy/roundrobin"
+	// "github.com/spf13/viper"
```
Comment (ldez) [inline traefik.go]: missing comment removing ?

Comment (vdemeester) [inline traefik.go]: damned, you're too quick @ldez :stuck_out_tongue_closed_eyes: 
I was looking for gifs.. :sweat_smile:

Comment (EmileVauge) [inline traefik.go]: Ops, was also working on viper, damn multi-threaded programming...

**Comment by vdemeester** [comment]
@EmileVauge needs a rebase already :sweat:

**Comment by EmileVauge** [comment]
[Circle is dead](http://status.circleci.com/)

**Comment by vdemeester** [comment]
alive-alive

LGTM 🐴

**Pull request merged**